docusaurus.config.js

```diff
@@ -46,6 +46,21 @@ module.exports = {
                 sizes: 'any',
             },
         },
+        // Segment analytics
+        process.env.SEGMENT_TOKEN && {
```

Comment:
@katzino How is the `SEGMENT_TOKEN` env var used here? 🤔 I don't see it anywhere in the script 😄